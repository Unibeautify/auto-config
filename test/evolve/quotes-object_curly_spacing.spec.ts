import { testGenetic } from "../testGenetic";
import prettier from "@unibeautify/beautifier-prettier";
import jsBeautify from "@unibeautify/beautifier-js-beautify";

testGenetic("quotes + object_curly_spacing", {
  beautifiers: [prettier, jsBeautify],
  language: "JavaScript",
  originalText: `if(true){console.log({ hello: "world" });}`,
  desiredText: `if (true) {\n  console.log({hello: 'world'})\n}\n`,
});
