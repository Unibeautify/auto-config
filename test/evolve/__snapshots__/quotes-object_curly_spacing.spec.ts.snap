// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quotes + object_curly_spacing: beautifiedText 1`] = `
if (true) {
  console.log({hello: 'world'})
}

`;

exports[`quotes + object_curly_spacing: entity 1`] = `
Object {
  "options": Object {
    "beautifiers": Array [
      "Prettier",
    ],
    "end_with_semicolon": false,
    "object_curly_spacing": false,
    "quotes": "single",
  },
}
`;

exports[`quotes + object_curly_spacing: fitness 1`] = `2`;
